src/main.cc

```diff
@@ -564,6 +564,7 @@ mainHandleCommandLineOption(const int optId, const char *optValue)
             /** \li On interrupt send SIGINT. */
             opt_send_signal = SIGINT;
         else if (!strncmp(optValue, "kill", strlen(optValue)))
+            // XXX: In SMP mode, uncatchable SIGKILL only kills the master process
```

Comment:
I would be happy to remove this comment if it is controversial.

Reply:
What does the comment mean though?